src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -468,11 +468,46 @@ function parseBuildGradleFile(gradleFile: string) {
     throw new Error(`Unable to find gradle file "${buildGradlePath}".`);
   }
 
+  return buildGradlePath;
+}
+
+function parseBuildGradleFile(gradleFile: string) {
+  const buildGradlePath: string = getBuildGradleFilePath(gradleFile);
   return g2js.parseFile(buildGradlePath).catch(() => {
```

Comment:
why don't we await here? I think the behaviour should be the same as in `parseGradlePropertiesFile`.

Reply:
Leave it as it is. 😄